### ISSUE TITLE
Slider : offsetPct variable undefined

### ISSUE THREAD

**Issue by guoni** [zurb/foundation-sites#8900]
foundation-sites/js/foundation.slider.js

Since 6.2.2, the variable offsetPct is not defined

```
  _handleEvent(e, $handle, val) {
    var value, hasVal;
    if (!val) {//click or drag events
      e.preventDefault();
      var _this = this,
          vertical = this.options.vertical,
          param = vertical ? 'height' : 'width',
          direction = vertical ? 'top' : 'left',
          eventOffset = vertical ? e.pageY : e.pageX,
          halfOfHandle = this.$handle[0].getBoundingClientRect()[param] / 2,
          barDim = this.$element[0].getBoundingClientRect()[param],
          windowScroll = vertical ? $(window).scrollTop() : $(window).scrollLeft();
         **missing variable offsetPct**

      var elemOffset = this.$element.offset()[direction];

      // touch events emulated by the touch util give position relative to screen, add window.scroll to event coordinates...
      // best way to guess this is simulated is if clientY == pageY
      if (e.clientY === e.pageY) { eventOffset = eventOffset + windowScroll; }
      var eventFromBar = eventOffset - elemOffset;
      var barXY;
      if (eventFromBar < 0) {
        barXY = 0;
      } else if (eventFromBar > barDim) {
        barXY = barDim;
      } else {
        barXY = eventFromBar;
      }
      **offsetPct** = percent(barXY, barDim);
```

Best regards,
Guillaume

**Comment by abdullahsalem** [comment]
@guoni Could you please use this [issue template](https://github.com/zurb/foundation-sites/blob/develop/.github/ISSUE_TEMPLATE.md)

**Comment by jasonraimondi** [comment]
I am seeing the same issue. This is happening both when I am running through Gulp/babel (laravel elixir babel to be exact), and webpack.

**How can we reproduce this bug?**

1. Webpack build with babel preset to ['es2015']
2. Place slider on an html page
3. The slider renders fine, but does not slide at all, trying to slide the slider causes console errors
4. ![screen shot 2016-06-06 at 7 17 45 pm](https://cloud.githubusercontent.com/assets/5787967/15844001/787ebc22-2c1b-11e6-9d3a-aebb481a86d5.png)

**What did you expect to happen?**
The slider to slide

**What happened instead?**
Slider errors and nothing happens, tick doesnt move, console logs
![screen shot 2016-06-06 at 7 17 45 pm](https://cloud.githubusercontent.com/assets/5787967/15844001/787ebc22-2c1b-11e6-9d3a-aebb481a86d5.png)

**Comment by jasonraimondi** [comment]
the fix is simply initializing the offsetPct with `var offsetPct...`

**Comment by kball** [comment]
Fixed in #8948
